website/src/content/docs/docs/howtos/Generate client libraries/02package.mdx

```diff
@@ -90,7 +117,34 @@ namespace Customization;
 ```
 
 ```java
-// TODO
+// Client builder class
+package petstore;
+
+@ServiceClientBuilder(serviceClients = { PetStoreClient.class, PetStoreAsyncClient.class })
+public final class PetStoreClientBuilder implements HttpTrait<PetStoreClientBuilder>,
+    ConfigurationTrait<PetStoreClientBuilder>, EndpointTrait<PetStoreClientBuilder> {
+
+    public PetStoreClientBuilder();
+
+    public PetStoreClient buildClient();
+}
+
+// Client class
+@ServiceClient(builder = PetStoreClientBuilder.class)
+public final class PetStoreClient {
+    public void feed(Food food);
+}
+
+// Model class
+package petstore.models;
```

Comment:
In line
```
@@clientNamespace(PetStore.Food, "PetStore.Models");
```
https://github.com/Azure/typespec-azure/blob/c94d1d233e90c598bf5bf6309ae0804af5a9bb79/website/src/content/docs/docs/howtos/Generate%20client%20libraries/02package.mdx?plain=1#L104

For Java, the customization of client namespace be the same as default.